src/mscorlib/corefx/System/Globalization/CalendarData.cs

```diff
@@ -302,6 +302,65 @@ private void InitializeAbbreviatedEraNames(string localeName, CalendarId calenda
                     break;
             }
         }
+
+        internal static CalendarData GetCalendarData(CalendarId calendarId)
+        {
+            //
+            // Get a calendar.
+            // Unfortunately we depend on the locale in the OS, so we need a locale
```

Comment:
I believe this holds true for ICU as well - ICU requires a locale (or timezone) to create a Calendar